client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDatabaseMetaData.java

```diff
@@ -1821,6 +1930,13 @@ private Connection createConnection(String catalog, String schema)
         return DriverManager.getConnection(url, "admin", null);
     }
 
+    private Connection createConnection(String catalog, String schema, boolean assumeNullCatalogMeansCurrentCatalog)
```

Comment:
NIT: `createConnectionWithNullCatalogMeansCurrent`. the boolean on the caller side doesn't give any idea what's the difference without looking into the method.

Reply:
done